editoast/src/main.rs

```diff
@@ -86,6 +87,26 @@ async fn runserver(
     // Setup shared states
     let infra_caches = Data::new(CHashMap::<i64, InfraCache>::default());
 
+    // Setup sentry
+    let _guard = match (
+        args.sentry_dsn,
+        args.sentry_env,
+        env::var("OSRD_GIT_DESCRIBE").ok(),
```

Comment:
OSRD_GIT_DESCRIBE is optional

Reply:
I think it would be reasonable to require it to send data to sentry.